kolibri/plugins/coach/serializers.py

```diff
@@ -95,12 +99,15 @@ def get_progress(self, target_node):
             return progress
         else:
             # filter logs by a specific leaf node and compute stats over queryset
-            leaf_node_stats = ContentSummaryLog.objects \
+            leaf_node_stats_query = ContentSummaryLog.objects \
                 .filter(content_id=target_node.content_id) \
-                .filter(user__in=get_members_or_user(kwargs['collection_kind'], kwargs['collection_id'])) \
-                .aggregate(total_progress=Coalesce(Sum('progress'), V(0)),
-                           log_count_total=Coalesce(Count('pk'), V(0)),
-                           log_count_complete=Coalesce(Sum(Case(When(progress=1, then=1), default=0, output_field=IntegerField())), V(0)))
+                .filter(user__in=get_members_or_user(kwargs['collection_kind'], kwargs['collection_id']))
```

Comment:
All of these default parameters tell me that these values are optional in the query; where can I look to find what parameters are required, if there are any?